docs/scenarios_authoring/Spel.md

```diff
@@ -278,12 +279,20 @@ More usage examples can be found in [this section](#conversions-between-datetime
 
 Explicit conversions are available in utility classes and build-in java conversion mechanisms:
 
-| Expression                                                      | Result                     | Type            |
-|-----------------------------------------------------------------|----------------------------|-----------------|
-| `#NUMERIC.toNumber('42')`                                       | 42                         | Number          |
-| `#NUMERIC.toNumber('42').toString()`                            | '42'                       | String          |
-| `#DATE_FORMAT.parseOffsetDateTime('2018-10-23T12:12:13+00:00')` | 1540296720000              | OffsetDateTime  |
-| `#DATE_FORMAT.parseLocalDateTime('2018-10-23T12:12:13')`        | 2018-10-23T12:12:13+00:00  | LocalDateTime   |
+| Expression                                                      | Result                    | Type           |
+|-----------------------------------------------------------------|---------------------------|----------------|
+| `#NUMERIC.toNumber('42')`                                       | 42                        | Number         |
+| `#NUMERIC.toNumber('42').toString()`                            | '42'                      | String         |
+| `#DATE_FORMAT.parseOffsetDateTime('2018-10-23T12:12:13+00:00')` | 1540296720000             | OffsetDateTime |
+| `#DATE_FORMAT.parseLocalDateTime('2018-10-23T12:12:13')`        | 2018-10-23T12:12:13+00:00 | LocalDateTime  |
+| `'true'.toBoolean`                                              | true                      | Boolean        |
+| `'11'.toLong`                                                   | 11                        | Long           |
+| `'1.1'.toDouble`                                                | 1.1                       | Double         |
+| `2.1.toBigDecimal`                                              | 2.1                       | BigDecimal     |
```

Comment:
What about casting section? and can... is... methods? I think that we need to refactor this documentation part more holistic, to make it clear what is the relation between casting and conversions. I think that we need to describe the general convention concept of to.. to...OrNull, is... methods and point that e.g. toDouble is a shortcut for `to('Double')`

Reply:
> make it clear what is the relation between casting and conversions
In next version we won't have casting because it's not implemented for now so I removed casting section and refactored a bit `explicit conversion` section.
I would refactor this part of docs after adding casting.